internal/cli/usercmd/configuration.go

```diff
@@ -500,7 +500,7 @@ func transformForDisplay(v string) string {
 	// Consider: Count and truncate by runes, not bytes.
 	// - https://pkg.go.dev/unicode/utf8@go1.20.5#RuneCountInString
 	// - https://go.dev/blog/strings (Libraries, foreach/range)
-	limit := 30
+	limit := 70
```

Comment:
what is the meaning of this limit? oveflowing this value is something tested (or that makes sense to test) ?

Reply:
Values larger than that are shown truncated (*). See https://github.com/epinio/epinio/pull/2513/files#diff-db92da3c647e894be7caec46cf94133def888e5c77dbc5a1109a4b9ec6b1614f for an affected test (configurations).

(*) They are likely multi-line as well, something the table formatting will have trouble with.